.github/workflows/main.yml

```diff
@@ -37,11 +37,11 @@ jobs:
           uses: felixmosh/turborepo-gh-artifacts@v1
           with:
             repo-token: ${{ secrets.GITHUB_TOKEN }}
-            server-token: ${{ secrets.TURBO_SERVER_TOKEN }}
+            server-token: 05de0230f01174d1f8cb4845a01dc6c895ce28f04ebef2318ab11615791b871c35eabbf8
         - name: Build
-          run: pnpm build -- --api="http://127.0.0.1:9080" --token="${{ secrets.TURBO_SERVER_TOKEN }}" --team="foo"
+          run: pnpm build -- --api="http://127.0.0.1:9080" --token="05de0230f01174d1f8cb4845a01dc6c895ce28f04ebef2318ab11615791b871c35eabbf8" --team="foo"
         - name: Lint
-          run: pnpm lint -- --api="http://127.0.0.1:9080" --token="${{ secrets.TURBO_SERVER_TOKEN }}" --team="foo"
+          run: pnpm lint -- --api="http://127.0.0.1:9080" --token="05de0230f01174d1f8cb4845a01dc6c895ce28f04ebef2318ab11615791b871c35eabbf8" --team="foo"
```

Comment:
This is great! 

Only thing is that it looks like it would be better to provide the `TURBO_TOKEN` env variable instead of passing `--token` each time.

https://turborepo.org/docs/reference/command-line-reference#--token

Reply:
@flybayer I have updated the workflow.